modelina-website/src/components/playground/Playground.tsx

```diff
@@ -87,11 +87,14 @@ class Playground extends React.Component<
     tsIncludeJsonBinPack: false,
     csharpArrayType: 'Array',
     csharpAutoImplemented: false,
-    phpIncludeDescriptions: false,
     csharpOverwriteHashcode: false,
     csharpIncludeJson: false,
     csharpOverwriteEqual: false,
     csharpIncludeNewtonsoft: false,
+    csharpNamespace: '',
```

Comment:
Maybe we should default the value here as well 🤔 That way in the options its the default value you edit instead of an empty field 🤔

Reply:
Sure